packages/components/checkbox/src/checkbox.vue

```diff
@@ -1,32 +1,32 @@
 <template>
   <label
     :id="id"
-    class="el-checkbox"
     :class="[
-      checkboxSize ? 'el-checkbox--' + checkboxSize : '',
-      { 'is-disabled': isDisabled },
-      { 'is-bordered': border },
-      { 'is-checked': isChecked },
+      ns.b(),
+      checkboxSize ? ns.m(checkboxSize) : '',
```

Comment:
```suggestion
      ns.m(checkboxSize),
```
If checkboxSize is falsey, ns.m will return an empty string.

Reply:
ok